SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs

```diff
@@ -27,6 +27,7 @@ public void Setup()
         }
 
         [Test]
+        [Ignore("dbo.sccv_team does not exist")]
```

Comment:
please remove this ignore statement so the test runs as expected. Once merged, the table will be available to run the test